modules/convert/convert.go

```diff
@@ -257,7 +257,7 @@ func ToTeam(team *models.Team) *api.Team {
 
 // ToUser convert models.User to api.User
 // signed shall only be set if requester is logged in. authed shall only be set if user is site admin or user himself
-func ToUser(user *models.User, signed, authed bool) *api.User {
+func ToUser(user *models.User, signed bool, authed bool) *api.User {
```

Comment:
This line doesn't need to change, it's valid Go.

Reply:
yes thought this is more readable

Reply:
@jolheiser yes thought this is more readable

Reply:
I agree, this is preffered:
```suggestion
func ToUser(user *models.User, signed, authed bool) *api.User {
```

Reply:
OK